wordpress-mobile/WordPress-Android
# Issue/remove lock unlock broadcastreceiver

Removes the lock/unlock broadcastreceiver and removes check for PIN lock being enabled to decide whether to add (or not add) quick actions to notifications.

Now all notifications are created once and all of them have quick actions, regardless of PIN lock feature being enabled.

To test:
1. Enable PIN lock by going to App Settings, and turning PIN lock on, then entering your PIN.
2. lock the phone screen
3. make a comment on a site you're an admin of
4. a notification should be received on the device
5. check that the notification has quick actions enabled.

cc @maxme @daniloercoli
